src/app/components/wrapper-components/rules-configuration/rules-configuration.component.spec.ts

```diff
@@ -14,13 +14,19 @@ describe('RulesConfigurationComponent', () => {
 
     let component: RulesConfigurationComponent;
 
-    async function chooseConfig(configIndex: number): Promise<void> {
+    async function chooseConfig(configId: string): Promise<void> {
         const selectAI: HTMLSelectElement = testUtils.findElement('#ruleSelect').nativeElement;
-        selectAI.value = selectAI.options[configIndex].value;
+        const option: HTMLOptionElement = Array.from(selectAI.options)
+            .find((opt: HTMLOptionElement) => opt.id === configId) as HTMLOptionElement;
+        selectAI.value = option.value;
```

Comment:
don't use `configId` but rather take `configName` as argument, and `selectAI.value = configName` + checker que le configName soit dans les éléments de l'option